java/test/jmri/jmrix/loconet/LnThrottleManagerTest.java

```diff
@@ -20,9 +21,12 @@
 public class LnThrottleManagerTest extends jmri.managers.AbstractThrottleManagerTestBase {
 
     private DccThrottle throttle;
+    private DccThrottle throttle2;
```

Comment:
Since this is only used in a single test, I think this should be a local variable in that test

Reply:
I envision more testcases using multiple-throttle setups, so made the "second" copies "fields".  But there's no good reason why they cannot be test-specific local variables.

Reply:
Actually, since these variables are assigned values within the listener, they need to be fields, not local variables.  (At least I can't get code to compile if I move the declarations into the tests.)